internal/vds/vds_test.go

```diff
@@ -425,6 +425,61 @@ func TestFence(t *testing.T) {
 	}
 }
 
+func TestFenceBorders(t *testing.T) {
+	testcases := []struct {
+		coordinate_system int
+		coordinates       [][]float32
+		interpolation     string
+		error             string
+	}{
+		{
+			coordinate_system: CoordinateSystemAnnotation,
+			coordinates:       [][]float32{{3, 10}, {6, 11.25}},
+			interpolation:     "nearest",
```

Comment:
Does the interpolation method affect the result in any way? Why does it need to be part of the test case?

Reply:
Then I would say that the added complexity/noice outweighs the benefit here. The more variables you add to tests, the less obvious it is what you are testing for